README.md

```diff
@@ -196,15 +196,16 @@ We thank EleutherAI for their work on the [lm-evaluation harness](https://github
 ## Cite as
 
 ```
-@software{bigcode-evaluation-harness,
+@misc{bigcode-evaluation-harness,
```

Comment:
The current citation format throws an error in LaTeX; This is the same format that lm-eval-harness uses